Update Bluetooth guide (#1867)
Samsung/tizen-docs: docs/application/native/guides/connectivity/bluetooth.md

  You can use LE operations to [manage scans](#le_scan), [set scanning filters](#le_scan_filter), [add advertising data](#add_adv_data), [set the advertising connectable mode](#set_adv_conn), [set the advertising mode](#set_adv_mode), and [start and stop advertising](#start_adv).

>   You can use extended advertising functionalities such like setting the primary phy, secondary phy and related scan functionalities since tizen 8.0.

Reviewer: ```suggestion
  > [!NOTE]
  > You can use extended advertising functionalities such as setting the primary phy, secondary phy, and related scan functionalities since Tizen 8.0.
```

   }
   ```

> ## Managing Bluetooth LE Extended Scans

Reviewer: ```suggestion
## Manage Bluetooth LE extended scans
```

> To discover nearby extended advertising packet, perform an LE extended scan operation:

Reviewer: ```suggestion
To discover nearby extended advertising packets, perform an LE extended scan operation:
```

1. To start the LE extended scan:

   ```
   int
   main()
   {
       int ret = BT_ERROR_NONE;
       ret = bt_adapter_le_start_scan_new(__bt_adapter_le_new_scan_result_cb, NULL);

       if (ret != BT_ERROR_NONE)
           dlog_print(DLOG_ERROR, LOG_TAG, "[bt_adapter_le_start_scan_new] failed.");

       /*
          Wait while the system searches for the LE target you want to connect to
          When you find the LE target you want, stop the LE scan

          bt_adapter_le_start_scan_new() operates continually
          until you call bt_adapter_le_stop_scan()
          If you do not call bt_adapter_le_stop_scan() after calling
          bt_adapter_le_start_scan_new(), calling bt_adapter_le_start_scan_new() again
          can cause an in-progress error
       */

       ret = bt_adapter_le_stop_scan();
       if (ret != BT_ERROR_NONE)
           dlog_print(DLOG_ERROR, LOG_TAG, "[bt_adapter_le_stop_scan] failed.");

       return;
   }
   ```

> 2. Use the callback defined in the `bt_adapter_le_start_scan_new()` function to retrieve the handle of scan results. The callback contains the handle on the scanned devices. you can retrieve the primary phy, secondary phy, advertising sid, periodic advertising interval, and the legacy information such as the device names, scanned devices' transmission level, service data list, appearance of the devices, and manufacturer data of the devices.

Reviewer: ```suggestion
2. Use the callback defined in the `bt_adapter_le_start_scan_new()` function to retrieve the handle of scan results. The callback contains the handle on the scanned devices. You can retrieve the primary phy, secondary phy, advertising sid, periodic advertising interval, and the legacy information such as the device names, scanned devices' transmission level, service data list, appearance of the devices, and manufacturer data of the devices.
```
